src/main/java/weavers/siltarae/tag/domain/Tag.java

```diff
@@ -35,5 +39,13 @@ public class Tag {
 
     private LocalDateTime deletedAt;
 
-
+    @Builder
+    public Tag(Long id, String name, User user, List<Mistake> mistakes, LocalDateTime createdAt, LocalDateTime deletedAt) {
+        this.id = id;
+        this.name = name;
+        this.user = user;
+        this.mistakes = mistakes;
+        this.createdAt = createdAt;
+        this.deletedAt = deletedAt;
+    }
 }
```

Comment:
@AllArgsConstructor 를 사용안하신 이유가 궁금합니다

Reply:
그렇다면 deletedAt 필드를 생성자에서 제외하고, deletedAt을 업데이트하는 별도의 메서드를 Entity 내에 만들어주는건 어떨까요?
현재 상태로는 
- deletedAt 필드가 채워져있고
- 기존 저장된 데이터와 아이디가 동일한 객체

를 생성한 뒤 save하면 기존의 데이터가 삭제 처리될 위험이 있을 것 같아요.

```suggestion
    @Builder
    public Tag(Long id, String name, User user, List<Mistake> mistakes, LocalDateTime createdAt) {
        this.id = id;
        this.name = name;
        this.user = user;
        this.mistakes = mistakes;
        this.createdAt = createdAt;
    }
}
```